README.md

```diff
@@ -1 +1,56 @@
-The start of something great
+# CloudQuery Onfleet Source Plugin
+
+[![test](https://github.com/cloudquery/cq-source-onfleet/actions/workflows/test.yaml/badge.svg)](https://github.com/cloudquery/cq-source-onfleet/actions/workflows/test.yaml)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cloudquery/cq-source-onfleet)](https://goreportcard.com/report/github.com/cloudquery/cq-source-onfleet)
+
+An Onfleet source plugin for CloudQuery that loads data from Onfleet to any database, data warehouse or data lake supported by [CloudQuery](https://www.cloudquery.io/), such as PostgreSQL, BigQuery, Athena, and many more. Based on a fork of (MIT LIcense https://github.com/keplr-team/go-onfleet). 
```

Comment:
The Keplr Go Onfleet API wrapper is out-of-date

Reply:
Done :)